### PR TITLE
Issue #12080: Github action Update Github milestone and upload '-all'  jar

### LINKED ISSUE #12080: Github action "Update Github"
https://github.com/checkstyle/contribution/blob/master/releasenotes-builder/diagrams-manual-trigger.md

we need to make action to do `run Github action "Update Github '10.3.1'"` (action takes human provided version as input, just to make sure it can not be executed unintentionally)
if user provider input is not matching pom - fail build

so we simply make script that update github milestones and upoad "-all.jar"
see all code in release.sh

### PR DESCRIPTION
**Pull request by Rahulkhinchi03**
Fixes #12080

### CODE REVIEW

**Comment by Rahulkhinchi03** [comment]
@romani How can I test this? As the script needs an input of the latest version because it will check latest and target. So if I do 10.3.1 then it gives error and if I use 10.3.3 then it doesn't run github tag page script.

File: .ci/update-github.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e
+
+checkForVariable "GITHUB_TOKEN"
```
Comment (romani) [inline .ci/update-github.sh:5]: Did you run this ?
You forgot source of util.sh

File: .github/workflows/update_github.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Target Version without (-SNAPSHOT)'
+        required: true
+jobs:
+  github:
+    name: Update Github ${{ github.event.inputs.version }}
+    runs-on: ubuntu-latest
```
Comment (romani) [inline .github/workflows/update_github.yml:15]: >CURRENT_VERSION=$(xmlstarlet

Please install xmlstartlet

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: .ci/update-github.sh (hunk 1 above)
Comment (Rahulkhinchi03) [reply .ci/update-github.sh:7]: Did, Maybe I unintentionally removed it while pushing.  Done Changes

File: .github/workflows/update_github.yml (hunk 2 above)
Comment (Rahulkhinchi03) [reply .github/workflows/update_github.yml:15]: Done

**Review by romani — APPROVED** [review]
ok to merge

**Review by romani — APPROVED** [review]
ok to merge

**Pull request merged**